lib/brain.js

```diff
@@ -2157,15 +2229,19 @@ Brain.prototype.isLowBalance = function isLowBalance () {
 
 Brain.prototype.nonCompliantTiers = function nonCompliantTiers (triggers, history, triggerTx) {
   const triggered = getTriggered(triggers, history, triggerTx)
+  const triggeredFormatted = _.map(o => o.customInfoRequestId ? { ...o, requirement: o.customInfoRequestId } : o, triggered)
   const getHighestSuspend = _.compose(_.get('id'), _.maxBy('suspensionDays'), _.filter({ requirement: REQUIREMENTS.SUSPEND }))
-  this.suspendTriggerId = getHighestSuspend(triggered)
 
-  const requirements = _.uniq(_.map(_.get('requirement'))(triggered))
+  this.suspendTriggerId = getHighestSuspend(triggeredFormatted)
+  this.triggers = triggeredFormatted
+
+  const requirements = _.uniq(_.map(_.get('requirement'))(triggeredFormatted))
   const unorderedTiers = _.isEmpty(requirements) ? [] : _.union(requirements, ['sms'])
-  const requiredTiers = _.sortBy(name => _.indexOf(name, ORDERED_REQUIREMENTS))(unorderedTiers)
 
-  this.requirementAmountTriggered = getLowestAmountPerRequirement(triggered)
+  // Custom trigger is identified by its uuid and should be pushed to last
+  const requiredTiers = _.sortBy(name => uuid.validate(name) ? Infinity : _.indexOf(name, ORDERED_REQUIREMENTS))(unorderedTiers)
```

Comment:
Doesn't this lead to a scenario where custom triggers 'draw' on the sorting function and custom triggers being unordered between themselves? It's this is not an important scenario, just ignore it